python/tvm/dlight/gpu/decode_gemv.py

```diff
@@ -220,8 +220,7 @@ def _sch_inner_spatial(
         s = sch.fuse(*s)
         sch.reorder(s, r)
         if unroll_spatial_factor:
-            s, inner = sch.split(s, factors=[None, unroll_spatial_factor])
-            sch.reorder(s, r, inner)
+            s, _ = sch.split(s, factors=[None, unroll_spatial_factor])
```

Comment:
We can do this as the order of thread-binding loops does not matter.

Reply:
and this implies the new loop order is `(s, inner, r)` which meets the requirement of CrossThreadReduction pass :)